js_modules/dagster-ui/packages/ui-core/src/assets/useAssetDefinitionFilterState.tsx

```diff
@@ -126,7 +159,12 @@ export function filterAssetDefinition(
     }
   }
 
-  if (filters.computeKindTags?.length) {
+  const isAllComputeKindTagsSelected = filters.selectAllFilters?.includes('computeKindTags');
+  if (isAllComputeKindTagsSelected) {
+    if (!definition?.computeKind?.length) {
+      return false;
```

Comment:
Hmm, yeah I think I can change the useEffect that sets "selectAllFilters" to instead do that. But it's a little tricky. I'll do this in a follow up when I revisit this logic since I'm a little unsure and think there was some tricky edge cases